CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
   overrides,
 - Rename `course-detail__aside__run` selector to `course-detail__aside__runs`
   in `course_detail.html` template to better reflect its content.
+- Update .scss to reflect the change to `course-detail__aside__runs` and move
```

Comment:
I think you don't need to add this as the line above was not released yet.

Reply:
I removed it.